### ISSUE TITLE
BaldAvenger Plugins no longer supported on MACOS Ventura

### ISSUE THREAD

**Issue by Kxinho** [baldavenger/BaldavengerPlugins#6]
Hi everyone,

Everything is in the title.
BaldAvenger's plugins seem to have turned unsupported in MacOS Ventura.
Nothing related to DaVinci 18.6 or previous.

Anyone experiencing the same problem ?

**Comment by bentcowan** [comment]
Yes I have experience this too. I am running resolve 18.6 and Ventura 13.6.1. Plugins show 'successfully loaded' in the the resolve plugin manager but do not show up in edit or color page effects list. Anyone have any thoughts or solutions?
